src/main/java/teammates/common/datatransfer/questions/FeedbackContributionQuestionDetails.java

```diff
@@ -86,7 +86,11 @@ public String getQuestionWithExistingResponseSubmissionFormHtml(boolean sessionI
                 Slots.RESPONSE_INDEX, Integer.toString(responseIdx),
                 Slots.FEEDBACK_RESPONSE_TEXT, Const.ParamsNames.FEEDBACK_RESPONSE_TEXT,
                 Slots.DISABLED, sessionIsOpen ? "" : "disabled",
-                Slots.CONTRIB_SELECT_FRAGMENTS_HTML, optionSelectFragmentsHtml);
+                Slots.CONTRIB_SELECT_FRAGMENTS_HTML, optionSelectFragmentsHtml,
+                "${equalShareHelp}", responseIdx == 0
+                                     ? "<span class=\"glyphicon glyphicon-info-sign\"></span>"
+                                           + " More info about the equal share scale"
+                                     : "");
```

Comment:
These few lines of code appear twice in this class, so perhaps we could extract out the `responseIdx == 0` onwards to a separate method. This also helps avoid the "custom indentation" problem.